### DISCUSSION TITLE
Neovim: copilot stop suggesting when switch mode

### DISCUSSION THREAD

**Discussion by merlinvn** [community/community#28221] [Copilot Conversations]
Copilot only do the suggestion on the first switch to insert mode. If i escape and switch back to insert mode, it stop suggesting.

Do you have any idea about this?

Thanks.

**Comment by nvima** [comment]
Same for me, someone can help?

**Reply by afonsofrancof** [reply]
Same for me. It just works once

**Reply by nvima** [reply]
i was able to fix it for me, but i don't know exactly what it was. i think copilot has problems with tab in neovim. i put it on CTRL + S.
maybe my config helps you
https://github.com/nvima/.dotfiles/blob/main/nvim/.config/nvim/after/plugin/lsp.lua

**Reply by afonsofrancof** [reply]
Thanks! I switched from TAB and it works now :)
